api/config/swagger.py

```diff
@@ -403,18 +403,87 @@ def schema_view_swagger(request):
-                    url='/api/meetup/{meetupId}/questions/',
+                    url='/api/meetups/{meetupId}/questions/',
                     action='GET',
                     fields=[
                         coreapi.Field(
                             name='meetupId',
                             required=True,
                             location='path',
                             description='Id of the meetup'
-                        )
+                        ),
+                        coreapi.Field(
+                            name='page',
+                            required=False,
+                            location='query',
+                            description='page number'
+                        ),
+                        coreapi.Field(
+                            name='page_limit',
+                            required=False,
+                            location='query',
+                            description='The limit of data displayed per page'
+                        ),
                     ],
                     description='Fetching questions on a specific meetup'
                 ),
+                'Upvote a question': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/upvote',
+                    action='PATCH',
+                    fields=[
+                        coreapi.Field(
+                            name='meetupId',
+                            required=True,
+                            location='path',
+                            description='Id of the meetup from which the question is got'
+                        ),
+                        coreapi.Field(
+                            name='questionId',
+                            required=True,
+                            location='path',
+                            description='Id of the question to be upvoted'
+                        )
+                    ],
+                    description='Upvoting a specific question'
+                ),
+                'Downvote a question': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/downvote',
+                    action='PATCH',
+                    fields=[
+                        coreapi.Field(
+                            name='meetupId',
+                            required=True,
+                            location='path',
+                            description='Id of the meetup from which the question is got'
+                        ),
+                        coreapi.Field(
+                            name='questionId',
+                            required=True,
+                            location='path',
+                            description='Id of the question to be downvoted'
+                        )
+                    ],
+                    description='Downvoting a specific question'
+                ),
+                'Get votes on a Question': coreapi.Link(
```

Comment:
The upvotes and downvotes of a question should be returned as part of the question's data. Defining another endpoint is unnecessary here. Just modify the endpoint for retrieving a specific question to return the upvotes and downvotes as well.

Reply:
Thanks for that @davenjeru ,
Let me update it

Reply:
Hello @davenjeru ,
Since we do not have an endpoint for getting a specific question. I have added the endpoint and appended the details to it

Reply:
Hello @davenjeru ,
Since we do not have an endpoint for getting a specific question on a meetup, I have added it and appended the votes to it. Is that okay?